packages/material/src/complex/index.ts

```diff
@@ -5,7 +5,9 @@ import {
 } from '@jsonforms/core';
 import MaterialArrayControlRenderer from './MaterialArrayControlRenderer';
 
-export default registerStartupRenderer(
+registerStartupRenderer(
     rankWith(3, isArrayObjectControl),
     MaterialArrayControlRenderer
```

Comment:
shouldn't this be connected?

Reply:
We're using the default export of the `MaterialArrayControlRenderer` which is connected.